packages/atchops/CMakeLists.txt

```diff
@@ -73,6 +76,24 @@ if(NOT ESP_PLATFORM)
         list(APPEND ATCHOPS_TARGETS_TO_INSTALL mbedcrypto) # since FetchContent imports the third-party library as if we wrote it ourselves, we also need to export it https://stackoverflow.com/questions/71541589/how-to-exporting-targets-fetched-with-fetchcontent
         list(APPEND ATCHOPS_MAKE_AVAILABLE MbedTLS) # make this available later on, when we make all modules available all at once
     endif()
+
+    # #########################################################
+	# 2B. uuid4
+	# #########################################################
+	find_package(uuid4 QUIET)
+	if(NOT uuid4_FOUND)
+		message(STATUS "[ATCHOPS] uuid4 not found, fetching from GitHub..")
+		FetchContent_Declare(
+			uuid4
+			URL https://github.com/atsign-foundation/uuid4/releases/download/v1.0.1/uuid4-v1.0.1.zip
```

Comment:
Do we (can we) have an automated way of keeping this updated?

Reply:
One solution comes to mind; CMake offers rich functionality. So what we could is have the URL and URL_HASH sit on a `.env` file or similar, and have GitHub Actions make PRs to edit that text file with the URL to the updated `.zip` whenever MbedTLS makes a release...

This sounds like a lot of engineering work, so before we commit to that, I've asked the question on the CMake Reddit: https://www.reddit.com/r/cmake/comments/19dqfvi/automate_fetchcontent_urlgit_url/